ignite/templates/typed/map/map.go

```diff
@@ -710,7 +710,7 @@ func typesCodecModify(replacer placeholder.Replacer, opts *typed.Options) genny.
 		content = replacer.ReplaceOnce(content, typed.Placeholder, replacementImport)
 
 		// Interface
-		templateInterface := `registry.RegisterImplementations((*sdk.Msg)(nil),
+		templateInterface := `registrar.RegisterImplementations((*sdk.Msg)(nil),
```

Comment:
Just curious, what is `registrar` the new pkg name? Because it is the Portuguese translation of the registration word

Reply:
Yeah, it's just the registry name